### PR TITLE
Add patch for numpy 1.14.0 test to fix 6.10.2 build

### PR DESCRIPTION
**Pull request by honnibal**
The previous PR went green without testing against numpy 1.14.0. This patch fixes a test failure.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,6 +18,8 @@ build:
   features:
     - vc9   # [win and py27]
     - vc14  # [win and py>=35]
+  patches:
```
Comment (rmax) [inline recipe/meta.yaml]: `patches` goes in the `source` section.

**Comment by honnibal** [comment]
Hmm, Appveyor says "the batch file cannot be found". Has there been a change to conda smithy and we need to rerender?

**Comment by rmax** [comment]
@conda-forge-admin Please rerender.

**Comment by honnibal** [comment]
@rmax Thanks --- green!

**Pull request merged**